SAMtags.tex

```diff
@@ -100,11 +100,11 @@ \section{Standard tags}
   {\tt PT} & Z & Read annotations for parts of the padded read sequence \\
   {\tt PU} & Z & Platform unit \\
   {\tt Q2} & Z & Phred quality of the mate/next segment sequence in the {\tt R2} tag \\
-  {\tt QT} & Z & Phred quality of the sample-barcode sequence in the {\tt BC} (or {\tt RT}) tag \\
+  {\tt QT} & Z & Phred quality of the sample barcode sequence in the {\tt BC} tag \\
   {\tt QX} & Z & Quality score of the unique molecular identifier in the {\tt RX} tag \\
   {\tt R2} & Z & Sequence of the mate/next segment in the template \\
   {\tt RG} & Z & Read group \\
-  {\tt RT} & Z & Barcode sequence (deprecated; use {\tt BC} instead) \\
+  {\tt RT} & ? & Reserved for backwards compatibility reasons \\
```

Comment:
I think that the type ("Z") should be kept for backwards compatibility reasons. 

Imagine that we had a validator that checked the types found in a file against the spec. Even though a RT is no longer in use, an old file that had RT tag could be either valid or invalid, and you could know that from the type. 

What's the value in "forgetting" the type of the reserved RT tag?

Reply:
Putting ? here is consistent with “it cannot be used for any new purpose”. But I take your point about the previously-expected type being useful historical information and have added it.